fibonacci.py

```diff
@@ -0,0 +1,33 @@
+def fibo_recur(n):
+
+    if n == 0:
+        return 0
+    
+    if n == 1:
+        return 1
+
+    if n == 2:
+        return 1
+
+    return fibo_recur(n-1) + fibo_recur(n-2)
+
+n = int(input())
+print(fibo_recur(n))
+
+
+def fibo_dp(n, dp=dict()):
+
+    if n == 0:
+        return 0
+
+    if n == 1 or n == 2:
+        return 1
+
+    if n in dp:
+        return dp[n]
+
+    dp[n] = fibo_dp(n-1, dp) + fibo_dp(n-2, dp)
+
+    return dp[n]
+
```

Comment:
There is no getting input for fibo_dp! You should implement the input function

Reply:
sorry, I'll check it as soon as possible.

Reply:
great job!